src/api/spec/controllers/staging/staging_projects_controller_spec.rb

```diff
@@ -105,6 +105,7 @@
           assert_select 'broken_packages', 1 do
             assert_select 'entry', 2
           end
+          assert_select 'history', 1
```

Comment:
Is there a way to check if it's really what we expect? Now, it's only checking that there is a history element.

Reply:
Looking the other tests, I don't see value in testing active record and/or active view.. any suggestion @dmarcoux ?

Reply:
I'm not sure if we need to go further. It's true that this would test ActiveRecord / ActionView.